crates/context/src/local.rs

```diff
@@ -0,0 +1,91 @@
+use bytecode::{CodeType, Eof};
+use context_interface::LocalContextTr;
+use primitives::{keccak256, Bytes, HashMap, B256};
+
+/// Local context that is filled by execution.
+#[derive(Clone, Debug, Default)]
+pub struct LocalContext {
+    /// Mapping of initcode hash that contains raw bytes ready for validation or status of validation.
+    ///
+    /// Used in EIP-7873 EOF - TXCREATE to fetch initcode by hash and cache its validation.
+    pub initcode_mapping: HashMap<B256, Initcode>,
+}
+
+impl LocalContextTr for LocalContext {
+    fn insert_initcodes(&mut self, initcodes: &[Bytes]) {
+        self.initcode_mapping = initcodes
+            .iter()
+            .map(|b| (keccak256(b), Initcode::new(b.clone())))
```

Comment:
TXCREATE access it by its hash that we need to calculate. Hash is needed so that factory is sure what is creating it is a binding between tx data and already deployed contract